### PR TITLE
Extract BM flow's model

### PR DESCRIPTION
**Pull request by amitdahan**
### 🔦 Summary
Gather all user configuration of the Flow's Model.
Before this PR, the user's configuration was hardcoded for our `flow-bm` template, now it's dynamic.

### 🗺️ Test Plan
`flow-bm` template tests.

### CODE REVIEW

File: packages/yoshi-flow-bm/src/createFlowBMModel.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    componentPath: componentPath,
+  }));
+
+  const methods = globFiles(methodsPattern).map(methodPath => ({
+    methodId: `${moduleId}.methods.${path.parse(methodPath).name}`,
+    methodPath: methodPath,
+  }));
+
+  const [moduleInitPath] = globFiles(moduleInitPattern);
+  console.log(globby.sync('translations', { cwd, absolute: true }));
```
Comment (deanshub) [inline packages/yoshi-flow-bm/src/createFlowBMModel.ts]: ?

File: packages/yoshi-flow-bm/src/renderModule.ts [hunk 2]
```diff
@@ -66,11 +44,17 @@ createModule({
...
-});`,
-  );
+});`;
+
+const renderModule = (model: FlowBMModel) => {
+  const tempPath = path.resolve(__dirname, '../tmp');
+  const filePath = path.join(tempPath, 'module.ts');
+
+  fs.mkdirSync(tempPath, { recursive: true });
+  fs.writeFileSync(filePath, generateModuleCode(model));
```
Comment (deanshub) [inline packages/yoshi-flow-bm/src/renderModule.ts]: not sure why but yoshi mostly uses `outputFileSync` instead of `writeFileSync`

Comment (amitdahan) [reply packages/yoshi-flow-bm/src/renderModule.ts]: Now we know! It obsoletes the `mkdirSync` part which simplified my code a lot :-D! 
Thanks!

**Comment by amitdahan** [comment]
Can you check my change to the entry part of the `webpack.config`?
Is this what you meant?

**Comment by amitdahan** [comment]
Ping @deanshub 👋

**Comment by deanshub** [comment]
awesome, looks way cleaner 🏅

**Pull request merged**